cairosvg/helpers.py

```diff
@@ -383,7 +383,10 @@ def size(surface, string, reference='xy'):
 
     for unit, coefficient in UNITS.items():
         if string.endswith(unit):
-            number = float(string[:-len(unit)])
+            try:
+                number = float(string[:-len(unit)])
+            except (ValueError, TypeError, IndexError) as e:
```

Comment:
No need: TypeError, IndexError and "as e"
`except ValueError:`

Reply:
Yeah, you are right, we can`t get the IndexError or TypeError exceptions, because the logic is based on UNITS dict.